schema/annotation-source.yaml

```diff
@@ -121,13 +128,17 @@ $defs:
       conclusion:
         $ref_curie: gks.core:Coding
         description: >-
-          The conclusion drawn from the statement proposition, direction, strength, and/or 
-          confidence score.
-      direction:
+          A term that describes a final conclusion about the Statement's subject, based on the statement 
+          proposition, confidence level, and/or evidence direction and strength assessments.  
+      evidence_direction:
         type: string
-        enum: [ "supports", "uncertain", "refutes" ]
+        enum: [ "supports", "uncertain", "disputes" ]
         description: >-
           The direction of this statement with respect to the target proposition.
+       evidence_strength: 
```

Comment:
As noted above, I proposed moving this 'strength' attribute form InformationEntity to Statement (given it was defined above in a way specific to Statements). The name 'evidence_strength indicates that this represents the collective strength of all evidence assessed (in the direction reported by the evidence_direction attribute) .

Calling this 'evidence_strength' instead of just 'strength' aligns with the overall VA paradigm and with the 'evidence_direction' attribute - and more clearly indicates that this describes the collective strength of the all evidence assessed (in the direction reported by the evidence_direction attribute).

Reply:
see above, please consider changing back to `strength` or just calling it `assertion_strength` since this is not a representation of the collective evidence. I think it should always be a given that the evidence supports the final assertion. If a given piece of evidence at any level does or does not support the statement above then that evidence item (or line) should have it's own attribute to indicate it's particular strength/role/direction in regards to parent evidence or statement and method it is applied to in it's specific circumstance.  Let's keep the qualification of the assertion and evidence independent and clear.

Reply:
I had an interesting call with Sharon Plon today, discussing the notion of statement strength. She was saying that trying to formalize strength for ACMG/AMP pathogenicity classifications isn't very reflective of reality. That many reporting labs look at the evidence, then make a judgement call based on their own understanding of the disease/gene variant domain that may not align with the classification rubric. The end classification–the statement–is really just the outcome of that decision making process. So what it actually means to be "likely" / "definitive" may vary much more wildly than what the guidelines recommend.

To that end it might make sense that this attribute *is not used* for SCV / VCV data; that the `conclusion` field captures the various qualitative classifications from which inference may be drawn downstream and the `confidence_score` field captures the quantitative confidence of the statement in support of / opposition to the target proposition.

Reply:
Hi Alex - to be sure I understand, this refers to whether we explicitly capture 'likely' vs 'definitive' as a confidence/evidence level in its own field for a given SCV statement, correct?  And you are proposing to not do this - to only capture evidence/confidence direction, and rely on the composed terms in the conclusion/classification field to report the level (e.g. 'likely pathogenic').
 
Based on your rationale Alex (that curators often use additional info / domain understanding / bias) besides an objective assessment of the evidence in determining likely vs definitive) - I agree that this would be a reason not to use an `evidence_level` field to capture  'likely' vs 'definitive' (because as you say this assessment is based on more than just the strength of the evidence).  But IMO we can and should capture this info in a `confidence_level` field - because this is meant for exactly what you describe - the curators confidence, which may be based on evidence and other info.
 
Furthermore,  I think the confidence level field is the dedicated place to put this info when available.  This is the primary place to put this info . . . if there is a specific way that the community/domain characterizes their final conclusion/classification using a single pre-composed term, we can include this as well.  But IMO we should always record the decomposed representation where we can.